### DISCUSSION TITLE
Starting 2 Athens instances on the same database ends up with one cleaning out the database

### DISCUSSION THREAD

**Discussion by agentydragon** [athensresearch/athens#946] [Archive]
This happens to me a bunch when developing. I leave my release version of Athens (as in, not built from HEAD) open, then in my terminal, do the `lein dev` and `yarn run electron .` dance. The dev Athens instance opens an empty database, and writes it into the path where my personal database is. The other Athens instance notices the database got overwritten. `index.transit` ends up containing the default (empty + `Welcome` page) database. Instead, we should do something sane, like detect it and refuse to start maybe.

**Comment by shanberg** [comment]
How about when running in dev mode, Athens forks your database into a dev-only db?

**Comment by jefftangx** [comment]
I do the same workflow as you @agentydragon, often using both release and development versions of Athens on the same DB path, but it doesn't ever to clear my DB. :thinking:

**Reply by solarkraft** [reply]
Even opening 2 instances of release Athens (beta 66, Appimage on Arch) causes this to happen for me. 
With a non-default database path the new instance creates its db but the old one stays in place and is reopened when you close the instances and start up a new one. 

When the original database is open every new instance will create its own new database at the default path, when it's not  one instance is started with the correct database and the cycle repeats. 

There aren't any unusual log messages on either start type.

**Comment by agentydragon** [comment]
I also use the default path (~/athens). I haven't tested behavior on other
paths.
